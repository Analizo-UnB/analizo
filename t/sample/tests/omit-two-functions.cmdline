egypt --omit say_bye,say_hello .
