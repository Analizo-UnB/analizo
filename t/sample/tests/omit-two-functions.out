digraph callgraph {
"main" -> "callback" [style=dotted];

}
